http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcRequestAuthenticator.java

```diff
@@ -461,4 +544,124 @@ private void addScopes(String scopes, Set<String> allScopes) {
             allScopes.addAll(Arrays.asList(scopes.split("\\s+")));
         }
     }
+
+    private String convertToRequestParameter(URIBuilder redirectUriBuilder, String redirectUri, String state, List<NameValuePair> forwardedQueryParams) throws JoseException, IOException {
+        redirectUriBuilder.addParameter(SCOPE, OIDC_SCOPE);
```

Comment:
Is this needed here?

Reply:
What happens if additional scopes have been configured?

Reply:
My understanding is that if we have additional scope values, then all those get added to the request object, only the openid scope is added to the URI in human readable text. This is based on the following text from the openid specs: 
```
Even if a scope parameter is present in the Request Object value, a scope parameter MUST always be passed using the OAuth 2.0 request syntax containing the openid scope value to indicate to the underlying OAuth 2.0 logic that this is an OpenID Connect request.
```
https://openid.net/specs/openid-connect-core-1_0.html#RequestObject